lib/data_update_scripts/20201019192035_backfill_registered_for_users.rb

```diff
@@ -0,0 +1,12 @@
+module DataUpdateScripts
+  class BackfillRegisteredForUsers
+    def run
+      users = User.where(registered: false).or(User.where(registered_at: nil))
+      users.find_each do |user|
+        next if user.created_at.nil? || user.invitation_accepted_at.nil?
```

Comment:
I don't think `user.created_at.nil?` would ever be true right?

Reply:
I don't think so... I added it to be extra cautious!  I'm happy to take that check out, @Zhao-Andy 😄

Reply:
👍 yeah I think taking it out makes more sense!

Reply:
Done, @Zhao-Andy!